BlazarUI/app/scripts/components/repo-build/Commit.jsx

```diff
@@ -0,0 +1,64 @@
+import React, {Component, PropTypes} from 'react';
+import moment from 'moment';
+import {Link} from 'react-router';
+
+class Commit extends Component {
+
+  renderTimestamp() {
+    const {timestamp} = this.props.commitInfo;
+    const formattedTime = moment(parseInt(timestamp, 10)).format('LT');
+
+    return (
+      <div className="commits-container--commit-timestamp">
+        {formattedTime}
+      </div>
+    );
+  }
+
+  renderContent() {
+    const {message, author, modified, added, removed, url} = this.props.commitInfo;
+    const {name} = author;
+    const filesChanged = modified.length + added.length + removed.length;
+
+    return (
+      <div className="commits-container--commit-wrapper">
+        <div className="commits-container--commit-content">
```

Comment:
BEM nitpick throughout the file, element vs modifier
```
 <div className="commits-container">
    <div className="commits-container__commit-content">
    <span className="commits-container__commit-message">
```
etc.

and for brevity I would suggest
```
 <div className="commits-container">
    <div className="commits-container__content">
    <span className="commits-container__message">
```